test/cpp/tensorexpr/test_simplify.cpp

```diff
@@ -3554,11 +3556,13 @@ TEST(Simplify, SimplifyEliminateZeroLengthAlloc) {
     // Finds right Alloc/Free.
     VarHandle x("x", kInt);
     VarHandle y("y", kInt);
+    BufHandle b1(x, {0}, kInt);
```

Comment:
I think `Var` used as base handles in `Buf`s need to have `kHandle` dtype (not `kInt)`. But maybe we could remove them here entirely? I.e. replace
```
    VarHandle x("x", kInt);
    BufHandle b1(x, {0}, kInt);
```
with
```
    BufHandle b1("x", {0}, kInt);
```